driver-core/src/main/com/mongodb/operation/AsyncChangeStreamBatchCursor.java

```diff
@@ -42,7 +42,7 @@
                                  final AsyncReadBinding binding) {
         changeStreamOperation.startOperationTimeForResume(binding.getSessionContext().getOperationTime());
         this.changeStreamOperation = changeStreamOperation;
-        this.resumeToken = changeStreamOperation.getResumeToken();
+        //this.resumeToken = changeStreamOperation.getResumeToken();
```

Comment:
ChangeStreamOperation no longer has a `getResumeToken()` method. I commented this out to get the async code to compile. When I start on the async changes, I will address this.

Reply:
So this mean that change streams are broken for async until we address this?  If so, it means we can't push to master until we fix async.  Can we change this go call the getResumeAfter method instead so that async still works?

Reply:
I have changed this to call `getResumeAfter()`. Async tests work with this change.
Evergreen patch: https://evergreen.mongodb.com/version/5d041dc03066157773cdc82a